inst/schema/Column.schema.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "column_id": {
+      "type": "string"
+    },
+    "label": {
+      "type": "string"
+    },
+    "values": {
+      "oneOf": [
+        { "$ref": "NestedFilterOption.schema.json" },
```

Comment:
If `ColumnValue` is a rebrand of `FilterOption`, would it be more consistent to also have `NestedColumnValue` or similar?